frontend_lib/src/component/Timeline/CommentFilePreview.jsx

```diff
@@ -77,9 +84,13 @@ export class CommentFilePreview extends React.Component {
                   {props.apiContent.created}
                 </div>
 
-                <div
+                <a
                   className={classnames(`${props.customClass}__body__text`, 'comment__body__text')}
                   title={title}
+                  href={fileDownloadUrl}
+                  target='_blank'
```

Comment:
Do you actually need `target='_blank'`? If you don't I'd rather remove it.

Reply:
At first I copied all the attributes of the app file download link, but as it seems the only important part is `force_download=1` in the url, I can try to remove it.

Reply:
Changed.